feeder/src/vote.ts

```diff
@@ -295,33 +295,82 @@ interface VoteArgs {
 }
 
 export async function vote(args: VoteArgs): Promise<void> {
-  const client = new LCDClient({
-    URL: args.lcdAddress,
+  const maxLCDIndex = args.lcdAddress.length - 1
+  let currentLCDIndex = 0
+
+  let client = new LCDClient({
+    URL: args.lcdAddress[currentLCDIndex],
     chainID: args.chainID,
     gasPrices: args.gasPrices,
   })
   const rawKey: RawKey = await initKey(args.keyPath, args.password)
   const valAddrs = args.validator || [rawKey.valAddress]
   const voterAddr = rawKey.accAddress
-  const wallet = new Wallet(client, rawKey)
+  let wallet = new Wallet(client, rawKey)
 
   while (true) {
     const startTime = Date.now()
 
-    await processVote(client, wallet, args.source, valAddrs, voterAddr).catch((err) => {
-      if (err.isAxiosError && err.response) {
-        console.error(err.message, err.response.data)
-      } else {
-        console.error(err.message)
-      }
-
-      resetPrevote()
-    })
+    try {
+      await processVote(client, wallet, args.source, valAddrs, voterAddr)
+    } catch (err) {
+      console.log(`vote: lcd client unavailable, rotating to next lcd client: ${err}`)
```

Comment:
hmm can we filter the server internal error from the unreachable error?

Reply:
I changed it. It's filtered now the same way as another part of vote.ts